compiler/installer.ini

```diff
@@ -15,6 +15,7 @@ Platforms: """
   dragonfly: i386;amd64
   haiku: i386;amd64
   android: i386;arm;arm64
+  nintendoswitch: armv8a57
```

Comment:
I think this is only necessary if you are going to be compiling Nim on the Nintendo Switch. I'm guessing that won't happen, will it?

Reply:
Well, you never know. Linux can run on the switch already :)